quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ManagementDistTest.java

```diff
@@ -120,10 +121,25 @@ void testManagementDifferentRelativePath(LaunchResult result) {
         assertRelativePath(result, "/management");
     }
 
+    @Test
+    @Launch({"start-dev", "--http-relative-path=/auth", "--http-management-relative-path=/management"})
+    void testManagementRootRedirects(LaunchResult result, KeycloakDistribution distribution) {
+        assertRelativePath(result, "/management");
+
+        distribution.setRequestPort(8080);
+
+        when().get("/").then().statusCode(200).body(containsString("Welcome to Keycloak"));
+        when().get("/auth").then().statusCode(200).body(containsString("Welcome to Keycloak"));
```

Comment:
It'd be better to actually check the redirect happened. This would pass even if the content was served on `/auth` without the redirect.

Reply:
@vmuzikar Not sure, if I follow. Could you please elaborate a little bit more? Or you mean to check whether the 301 HTTP response code is present?

This use case tests the redirect by accessing the `get("/")`. In the old behavior, the request would end up with the 'Resource not found', because it's served on the relative path `/auth`.

> This would pass even if the content was served on /auth without the redirect.

The `get("/auth")` would pass in that case, but not the previous assert with the `get("/")`.

Reply:
> Or you mean to check whether the 301 HTTP response code is present?

Yes. My point was that there could be theoretically a bug that we'd serve the welcome screen from both `/`  and e.g. `/auth` instead of redirecting `/` to `/auth`.

Reply:
@vmuzikar Not sure how I can do that with the REST assured - it shows me only the final response -> 200 OK. Do you know how can I verify it? But it should be verified in some WelcomePage test that it's serving properly right?

Is this a blocker for you?

Reply:
Seems it [should be possible](https://stackoverflow.com/questions/74333165/how-can-i-turn-on-automatically-follow-redirect-in-rest-assured-java) to disable redirects following in REST assured.

Reply:
ok